koku/reporting/migrations/0214_auto_20220112_0209.py

```diff
@@ -0,0 +1,135 @@
+# Generated by Django 3.2.11 on 2022-01-12 02:09
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("reporting", "0213_delete_mat_views")]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Getting rid of all of the TextFields was a requirement for DJ3.2?

Reply:
I started typing a big response here, but I just thought of something. Lemme try this without a migration. Originally, I couldn't figure out a way, but I might know one now.

Reply:
As to why I did change this field, I kept seeing these errors on Azure tests:
```django.core.exceptions.FieldError: Expression contains mixed types: TextField, CharField. You must set output_field.```

This error is the result of this ([copied from the 3.2 release notes](https://docs.djangoproject.com/en/4.0/releases/3.2/#models)):

> Value() expression now automatically resolves its output_field to the appropriate Field subclass based on the type of its provided value for bool, bytes, float, int, str, datetime.date, datetime.datetime, datetime.time, datetime.timedelta, decimal.Decimal, and uuid.UUID instances. As a consequence, resolving an output_field for database functions and combined expressions may now crash with mixed types when using Value(). You will need to explicitly set the output_field in such cases.

Reply:
I think I figured out how to fix everything by updating the provider_maps instead of requiring a migration. We'll see how the testing goes...